tools/tplgtool2.py

```diff
@@ -739,7 +739,7 @@ def print_pcm_info(self):
     @cached_property
     def coreids(self):
         "All available core IDs."
-        cores = set()
+        cores = set([0])
```

Comment:
I found the code quite cryptic. I think these two changes would have really helped:
- A very short comment
- Not using a [ ] list indirection which looks like it's a set of lists.
```suggestion
        # Core 0 is the default value
        cores = {0}
```

Reply:
Ack